lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -0,0 +1,69 @@
+    And the user remains in the Accuracy of the appellant's submission sub-section
+
+  Scenario: AC03 LPA user selects Yes and proceeds to task list
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects Yes
+    And the user selects Save and Continue
+    Then the user continues to the Task List
+    And a 'Completed' status is populated on that sub-section of the task list
+
+  Scenario: AC04 LPA user selects no and is presented with free text field
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects No
+    Then the user is provided with a free text field to input their reasons
+
+  Scenario Outline: AC05 LPA user provides further information in text field and proceeds to task list
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects No
+    And the user enters '<inaccuracy_reason>'
+    And the user selects Save and Continue
+    Then the user continues to the task list
+    And a 'Completed' status is populated on that sub-section of the task list
+    Examples:
+      | inaccuracy_reason  |
+      | Field is incorrect |
+      | 1234567890 |
+      | This is a really long paragraph. Here is the rest of the really long paragraph. All these words are in this one paragraph. |
+      | Reason 1, Reason 2, Reason 3, Reason 4 |
+
+  Scenario: AC06 LPA user does not provides further information in text field and is provided an error
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects Yes
+    And has not provided further information as text regarding their reasons
+    And the user selects Save and Continue
+    Then the user is shown the error message 'Enter details of why this does not accurately reflect the planning application'
+    And the user remains in the Accuracy of the appellant's submission sub-section
+
+  Scenario: AC07 Appeal details side panel
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user returns to the 'Tell us about any appeals in the immediate area' page from the Task List
+    Then the user can see the appeal details panel on the right hand side of the page
+
+  Scenario: AC08 Selects back
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects No
+    And the user selects the back link
+    Then the user navigates to the task list
+    And any information they have inputted will not be saved
+
+  Scenario: AC09 LPA user has completed the 'Review the accuracy of the appellant's submission' sub-section and returns to that sub section from the Task List
+    Given a user has completed the information needed on the accuracy of the appellant's submission sub-section
+    When the user returns to the sub-section from the Task List
```

Comment:
Can we change **When the user returns to the sub-section from the Task List** to **When the user returns to the extra conditions page from the Task List** as this will be recorded in the Cypress report and it is more readable.

Reply:
This is inconsistent with the ticket - that refers to the sub-sections consistently. Do we need to change this? If so need to discuss with Pippa